Embedded/MaxMix/MaxMix.ino

```diff
@@ -208,7 +208,10 @@ void ProcessPackage()
 
     // Switch to newly added item.
     if(settings.displayNewSession)
+    {
       itemIndex = index;
+      state = STATE_APPLICATION_NAVIGATE;
```

Comment:
There are now multiple modes, application and game, each one has multiple states and they are all tracked by the same "state" variable.  
This change assumes you are in application mode and would break if you are in game mode when a new application is added.

You need to check if the mode == MODE_APPLICATION.

Reply:
Yup didn't realise it used the same variable. Was under the assumption it was different since it has it's own index variable as well. Fixed.